website/client/libs/store/helpers/orderTasks.js

```diff
@@ -0,0 +1,34 @@
+import compact from 'lodash/compact';
+import isEmpty from 'lodash/isEmpty';
+
+// sets task order for single task type only.
+// Accepts task list and corresponding taskorder for its task type.
+export function orderSingleTypeTasks (rawTasks, taskOrder = []) {
+  // if there is no predefined task order return task list as is.
+  if (isEmpty(taskOrder)) return rawTasks;
```

Comment:
at this point it's not necessary to pass a default to taskOrder, let's remove that and simply do `if (!taskOrder)`

Reply:
updated as suggested